src/LanguageServer.Engine/CompletionProviders/CommentCompletionProvider.cs

```diff
@@ -53,7 +53,7 @@ public CommentCompletion(ILogger logger)
         /// <returns>
         ///     A <see cref="Task{TResult}"/> that resolves either a <see cref="CompletionList"/>s, or <c>null</c> if no completions are provided.
         /// </returns>
-        public override async Task<CompletionList> ProvideCompletions(XmlLocation location, ProjectDocument projectDocument, string triggerCharacters, CancellationToken cancellationToken = default)
+        public override async Task<CompletionList> ProvideCompletionsAsync(XmlLocation location, ProjectDocument projectDocument, string triggerCharacters, CancellationToken cancellationToken)
```

Comment:
Heh, yeah, another one of those style things (but I don't feel all that strongly about it).

I normally only use the `Async` suffix if there is also a non-async version.